delivery_carrier_typology/models/delivery_carrier.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class DeliveryCarrier(models.Model):
+    _inherit = "delivery.carrier"
+
+    shipping_means_ids = fields.Many2many(
+        "delivery.carrier.typology",
+        string="Shipping means",
+    )
+
+
+class DeliveryCarrierTypology(models.Model):
+    _name = "delivery.carrier.typology"
```

Comment:
@chafique-delli That should have been an exstension of carrier category no ?

Reply:
yes, why not

Reply:
@rousseldenis , According to you, the 'category_id' field would be used to indicate a means of transport used by the delivery carrier ?

Reply:
Ok, it is more clear.